packages/api-explorer/i18n/en.json

```diff
@@ -28,8 +28,10 @@
   "api.response.binary": "A {typeFile} was returned. Download file to view it",
   "schemaTabs.missing.response": "Response schema not set",
   "schemaTabs.missing.request": "Request schema not set",
+  "schemaTabs.missing.example": "Example schema not set",
```

Comment:
"Example schema"? Isn't it an example request/response? Let's just write "No example set" ? (same applies for it translation)